client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -593,10 +592,6 @@ export default {
         }
 
         return dpApi.patch(Routing.generate('api_resource_update', { resourceType: 'Statement', resourceId: statementId }), {}, payload)
-          .then(response => {
-            checkResponse(response)
-            return response
-          })
           .then(response => {
             dplan.notify.notify('confirm', Translator.trans('confirm.statement.assignment.assigned'))
```

Comment:
This should be passed as `options` to the `dpApi` call

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2538/commits/30f745dac90d874f1494d04489aadc4d0c7b6668